ui/src/components/SubDescription/SubDescription.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import styled from 'styled-components';
+import { z } from 'zod';
+import { variables } from '@splunk/themes';
+import { SubDescriptionSchema } from '../../types/globalConfig/pages';
+
+export const SubTitleComponent = styled.div.attrs({
+    className: 'pageSubDescription',
+})`
+    &.pageSubDescription {
+        font-size: ${variables.fontSize};
+        margin-bottom: 10px;
+    }
+`;
+
+type SubDescriptionProps = z.TypeOf<typeof SubDescriptionSchema>;
+
+const mapTextToElements = (props: SubDescriptionProps) => {
+    const splitedtextBySlugs: string[] | undefined = props?.text.split(/\]\]|\[\[/);
+
+    return splitedtextBySlugs
+        ?.map((text, i) => {
+            const linkToReplace = props?.links?.find((link) => link.slug === text);
+
+            if (linkToReplace) {
+                return (
+                    <a // using index as key for elements as they dont have unique values but are unique itself
```

Comment:
why not `<Link>`?

Reply:
yea, forgot about that one once again ..., good catch